dora/core/common/src/main/java/alluxio/uri/UfsUrl.java

```diff
@@ -189,16 +230,38 @@ private UfsUrl(UfsUrlMessage proto) {
 
   /**
    * Constructs an {@link UfsUrl} from components.
-   * @param scheme the scheme of the path
+   *
+   * @param scheme    the scheme of the path
    * @param authority the authority of the path
-   * @param path the path component of the UfsUrl
+   * @param path      the path component of the UfsUrl
    */
   public UfsUrl(String scheme, String authority, String path) {
```

Comment:
how can I construct a URL without an authority, using this constructor? Should I pass `null` for `authority`?

Reply:
just pass empty string "", like the UfsUrlTest does.

Reply:
Added in javadoc.